targets/actionscript/make.js

```diff
@@ -123,6 +123,55 @@ function getVerticalNameDefault() {
     return "null";
 }
 
+function getValidPlayFabActionScriptNamespacePrefix(datatype) {
+
+    var asIndividualNamespaces = datatype.classNameSpace.split('.');
+
+    var asNamespace = "com.";
+
+    if (asIndividualNamespaces.length < 2)
+    {
+        throw new Error("Error in Generating API Model: Namespaces are expected to be in the form playfab.[SDK].[Name]");
+    }
+
+    for (var i = 0; i < asIndividualNamespaces.length; i++) {
+        if (i < 1) {
+            asNamespace += asIndividualNamespaces[i].toLowerCase();
+        }
+        else {
+            asNamespace += asIndividualNamespaces[i];
+        }
+
+        // we expect the input to be playfab.[SDK].[Name]
+        // but the actual datatype-callable prefix namespace in ActionScript is com.playfab.[SDK][Name].
+        if (i < asIndividualNamespaces.length - 2) {
+            asNamespace += ".";
+        }
+    }
+
+    // Current use expects this before the datatype name
+    asNamespace += ".";
+
+    return asNamespace;
+}
+
+function fullNamespaceRequired(propName)
+{
+    // NOTE: As of PlayFab version 191029, some objects are not properly getting recognized by ActionScript
+    // Due to the language getting deprecated within a year, we are going to add any breaking objects to this list
+    // and add the full namespace to any new object that breaks actionscript comiplation here.
+    var typesThatNeedFullNamespace = [ "TreatmentAssignment" ];
+
+    for(var i = 0; i < typesThatNeedFullNamespace.length; i++)
```

Comment:
We have some list.contains() helpers in generate.ts
Search the project for an example usage and exact function name.